core/scss/components/lockup/_lockup--option-h.scss

```diff
@@ -0,0 +1,23 @@
+@charset 'UTF-8';
+
+.su-lockup--option-h {
+
+  .su-lockup__line2,
+  .su-lockup__line5 {
+    display: none;
+  }
+
+  .su-lockup__line4 {
+    line-height: 0.95em;
+    margin-bottom: -3px;
```

Comment:
I remember we all discussed this, so apologies if this is inline with what we decided. but did we want to use the margin mixin for consistency even when only specifying an individual side for margin? If so, `@margin();` should be used here.